dashboard/app/models/course_version.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: course_versions
+#
+#  id                :integer          not null, primary key
+#  key               :string(255)      not null
+#  display_name      :string(255)      not null
```

Comment:
For lesson group we put display_name in as a property. Not sure if we should keep that consistent here?

Reply:
I was thinking we want every CourseVersion to have a display name, so it should be a column. I think this is slightly simpler than having optional display names and defaulting to using the key when not present. I don't feel strongly, though.

Any opinions @davidsbailey? If neither of you feel strongly on this, I'll keep it as is.

Reply:
I don't feel strongly, but the fact that you want the non-null requirement is enough to convince me to leave it as a column.

From an i18n perspective, the pattern of storing the english version translatable strings in the database aligns with what @Hamms has requested for new models. Anecdotally, with lesson groups, we tried to just store the translation in the .yml (or maybe now .json) files, and we ran into problems because those don't update fast enough, causing certain validations, which we wrote, to fail.

Ideally, when fetching "displayable" content in order to show it to the user, we should always use a helper method, in this case perhaps localized_display_name. That will make it easier for FND to come and update those methods to return translated results. I've previously attempted to capture the requirements for this here: https://codedotorg.atlassian.net/browse/PLAT-230

Reply:
Thanks, that helps a lot. Keeping it as-is, then.